# instance_name provides the name of the robot as is presented in the rcm platform
string instance_name

# instance_status provides the status of the robot in the rcm platform: CONNECTED=1 or DISCONNECTED=0
int16 instance_status

int16 CONNECTED=1
int16 DISCONNECTED=0
